utils/update-legacy-name.js

```diff
@@ -36,6 +36,10 @@ if (pkg.name === '@vercel/client') {
   pkg.name = 'now-client';
 } else {
   pkg.name = pkg.name.replace('vercel', 'now');
+  if (pkg.bin && pkg.bin.vercel) {
```

Comment:
Let's add a comment:

```suggestion

  // Rename the "vercel" binary to "now" for Now CLI
  if (pkg.bin && pkg.bin.vercel) {
```